src/index.js

```diff
@@ -2,25 +2,30 @@
 
 const bs58 = require('bs58')
 
-module.exports = PeerBook
-
-function PeerBook () {
-  if (!(this instanceof PeerBook)) {
-    return new PeerBook()
+class PeerBook {
+  constructor () {
+    this._peers = {}
   }
 
-  const peers = {}
-
-  this.put = (peerInfo, replace) => {
-    if (peers[peerInfo.id.toB58String()] && !replace) {
-      // peerInfo.replace merges by default
-      peers[peerInfo.id.toB58String()].multiaddr.replace([], peerInfo.multiaddrs)
+  /**
+   * Stores a peerInfo, if already exist, only adds the multiaddrs
```

Comment:
one more thing, it would be great if it could return the merged result, as I currently have to do this

```js
// got info from the network
peerBook.put(info)
const correctInfo = peerBook.get(info)

// after
const correctInfo = peerBook.put(info)
```